library/shared_library/clib4.c

```diff
@@ -495,12 +494,7 @@ struct Clib4Library *libInit(struct Clib4Library *libBase, BPTR seglist, struct
     /* Open resource */
     struct Clib4Resource *res = (APTR) iexec->OpenResource(RESOURCE_NAME);
     if (!res) {
-        res = iexec->AllocVecTags(
-                sizeof(struct Clib4Resource),
-                AVT_Type, MEMF_SHARED,
-                AVT_ClearWithValue, 0,
-                AVT_Lock, TRUE,
-                TAG_END);
+        res = calloc(1, sizeof(struct Clib4Resource));
```

Comment:
I tried to use mlock() to lock this memory, but unfortunately that is not available. Any idea on how to lock this memory is welcome.